packages/runtime/container-runtime/src/test/blobManager.spec.ts

```diff
@@ -121,7 +121,7 @@ export class MockRuntime
 				});
 				this.unprocessedBlobs.add(blob);
 				this.emit("blob");
-				this.blobPs.push(P.catch(() => {}));
+				this.blobPs.push(P.then(() => {}));
```

Comment:
```suggestion
				this.blobPs.push(P);
```

Reply:
then the fix for the below issues is probably to re-add the catch in the processes call, so that doesn't fail

Reply:
I have tried removing the then part, but by doing it onUploadResolve is not run after resolving blobPs. This causes that at this point we never send the attach blob op, therefore never being able to resolve the blob handle.